### ISSUE TITLE
share install on MacOS GitHub Actions gives unexpected library-load failure

### ISSUE THREAD

**Issue by mohawk2** [shawnlaffan/perl-alien-proj#20]
E.g. in https://github.com/PDLPorters/PDL-Transform-Proj4/actions/runs/12134943895/job/33833045527

```
  Can't load '/Users/runner/work/PDL-Transform-Proj4/PDL-Transform-Proj4/blib/arch/auto/PDL/Transform/Proj4/Proj4.bundle' for module PDL::Transform::Proj4: dlopen(/Users/runner/work/PDL-Transform-Proj4/PDL-Transform-Proj4/blib/arch/auto/PDL/Transform/Proj4/Proj4.bundle, 0x0002): Library not loaded: @rpath/libproj.25.dylib
    Referenced from: <F74CD300-5820-33A4-BEC2-74783C6D8410> /Users/runner/work/PDL-Transform-Proj4/PDL-Transform-Proj4/blib/arch/auto/PDL/Transform/Proj4/Proj4.bundle
  Error:   Reason: tried: '/usr/lib:/Users/runner/perl5/lib/perl5/darwin-2level/auto/share/dist/Alien-proj/lib/libproj.25.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/lib:/Users/runner/perl5/lib/perl5/darwin-2level/auto/share/dist/Alien-proj/lib/libproj.25.dylib' (no such file), '/usr/lib:/Users/runner/perl5/lib/perl5/darwin-2level/auto/share/dist/Alien-proj/lib/libproj.25.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/lib:/Users/runner/perl5/lib/perl5/darwin-2level/auto/share/dist/Alien-proj/lib/libproj.25.dylib' (no such file) at /Users/runner/hostedtoolcache/perl/5.30.3/arm64/lib/5.30.3/darwin-2level/DynaLoader.pm line 197.
   at t/proj_transform.t line 6.
```

Note in particular `tried: '/usr/lib:/Users/runner/perl5/lib/perl5/darwin-2level/auto/share/dist/Alien-proj/lib/libproj.25.dylib' (no such file)` - it's trying two dir names with a `:` in the middle, rather than splitting on the `:`.

**Comment by shawnlaffan** [comment]
Possibly it's related to Proj 9.5 which was released the other day.  I'll try to test it on my mac tomorrow.

I notice that the Alien::proj tests are currently disabled.  
https://github.com/PDLPorters/PDL-Transform-Proj4/actions/runs/12134943895/job/33833045527#step:4:73

It would be good to see if these pass or fail.

The deps can be installed without testing first to reduce noise (`cpanm -n --installdeps Alien::proj`).

**Comment by shawnlaffan** [comment]
I was able to reproduce this locally.  It is unrelated to the Proj version.

The root cause seems to be an interaction between Plugin::SearchDeps and system aliens (https://github.com/PerlAlien/Alien-Build/issues/419).  Alien::proj was filtering a system Alien::libtiff from the list, but not Alien::sqlite.  Now it does any system aliens that are passed (https://github.com/shawnlaffan/perl-alien-proj/pull/21).

I've released Alien::proj 1.30 to CPAN.  Can you give it a test?

**Comment by mohawk2** [comment]
It works great. Thank you very much!
